packages/create-cloudflare/templates-experimental/hello-world-with-assets/js/test/index.spec.js

```diff
@@ -1,4 +1,4 @@
-import { env, ctx, createExecutionContext, waitOnExecutionContext, SELF } from 'cloudflare:test';
+import { env, createExecutionContext, waitOnExecutionContext, SELF } from 'cloudflare:test';
 import { describe, it, expect } from 'vitest';
 import worker from '../src';
 
```

Comment:
also github won't let me comment on the relevant line, but I don't think we can pass `ctx` into SELF.fetch() on line 19 and 37 anymore.

Reply:
Thank you!